### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by nnrustamo**

### CODE REVIEW

**Review by larskotthoff — CHANGES_REQUESTED** [review]
Can you explain how the train/test split and the CV work together here? Please use BO for the optimization and a larger number of epochs for training the CNNs.

**Review by larskotthoff — CHANGES_REQUESTED** [review]
From the figures it looks like you used BO, but your report mentions grid search. Can you please be clear about what exactly you did and how?

**Comment by nnrustamo** [comment]
I updated the report.

**Review by larskotthoff — CHANGES_REQUESTED** [review]
Can you try BO please? This is a great case where BO should be better than grid search.

And please don't include the loss function in the space of tuned things -- this doesn't change anything about the fundamental model, just how it is measured.

**Comment by nnrustamo** [comment]
Dr. Kotthoff,

I have used BO for this project, please read the report and the look at the notebook that I last submitted. I have excluded loss function from the search space in the pipeline optimization and wildcard projects.
Can we please close this project?

Thank you,
Nijat

**Comment by larskotthoff** [comment]
Am I looking at the wrong file? [Hyper Parameter Optimization.pdf](https://github.com/COSC5557/hyperparameter-optimization-supersonic98/blob/main/Hyper%20Parameter%20Optimization.pdf) doesn't say anything about BO and includes the loss function in the search space.

**Comment by nnrustamo** [comment]
Maybe you are looking at the wrong branch?

**Comment by larskotthoff** [comment]
Can you give me a link to the correct file please, or make sure that there's not a bunch of old stuff in here? I had a look at another report in another branch, but that one doesn't discuss nested resampling and includes the loss functions in the HP search space.

**Comment by nnrustamo** [comment]
Here are the links to the report and the code file.
https://github.com/COSC5557/hyperparameter-optimization-supersonic98/blob/supersonic98-patch-1/Hyper%20Parameter%20Optimization.pdf
https://github.com/COSC5557/hyperparameter-optimization-supersonic98/blob/supersonic98-patch-1/hyper_parameter_optimization.ipynb

**Comment by larskotthoff** [comment]
And that report doesn't discuss nested resampling and has the loss functions listed in Table 1.

**Comment by nnrustamo** [comment]
I insist on keeping loss function in the search space (in this exercise only). Loss function is not just for the evaluation, the correct loss function is necessary to represent the correct physics. As a result of BO, top 10 best models all had mean squared logarithmic error as the loss function. So, in the next exercise I continue with mean squared logarithmic error and exclude it from the search space. Without including them in the search space I would never know that mean square logarithmic error performs better than, for example, mean squared error. I discussed this finding in the presentation as well. As per nested resampling, do you want me to rerun everything and submit it again with nested resampling?
